spec/spec_helper.rb

```diff
@@ -2,20 +2,30 @@
 Bundler.setup(:default, :development)
 require 'capybara/rspec'
 require 'capybara-screenshot/rspec'
+require 'pry'
```

Comment:
Add this in the Gemfile under development. We don't want to install pry in prod containers, you're better off using byebug which is an actual debugger and already included in the Gemfile.

Reply:
That should not be there - I'll remove